app/digests/api.py

```diff
@@ -46,6 +46,23 @@ def _create_response(self, data: Dict[str, Any]) -> Response:
     def _can_preview(self) -> bool:
         return self.request.META.get(settings.AUTHORIZATION_PREVIEW_HEADER, False)
 
+    @staticmethod
+    def _filter_by_related_content_id(qs: QuerySet, content_id: str) -> QuerySet:
+        """Will filter by `relatedContent` `id` value if present.
+
+        :param qs: class: `QuerySet`
+        :param content_id: str
+        :return: class: `QuerySet`
+        """
+        return qs.filter(
+            relatedContent__contains=[
+                {
+                    'id': content_id,
+                    'type': 'research-article'
```

Comment:
I'm not sure about this filter. Is it all the `type` we have to care about? Because [all these are articles](https://github.com/elifesciences/api-raml/blob/develop/src/snippets/article.v1.yaml#L13-L25).

Reply:
Should probably be passed from the outside? Or be part of the query `?relatedContaining=research-article/42`? API design decisions.

Reply:
I like the passed from outside concept, it would make it easier to adapt to a query parameter later if required, otherwise it looks like only articles of type ``research-article`` have digests right now when I search the archives.